src/cephadm/cephadm.py

```diff
@@ -4538,6 +4541,12 @@ def _add_deploy_parser_args(
         default=[],
         help='Additional entrypoint arguments to apply to deamon'
     )
+    parser_deploy.add_argument(
+        '--termination-grace-period-seconds',
+        type=int,
+        default=None,
+        help='Time in seconds to wait for graceful service shutdown before forcefully killing it'
```

Comment:
Some confusion may arise as to the behavior of `--termination-grace-period-seconds 0` -- does it mean proceeding with a forceful kill immediately or waiting for a graceful shutdown indefinitely?  Since the implementation boils down to `podman stop --timeout`, one can determine that it's the former, but it would be nice to mention that in the description here.